src/pages/patientView/PatientViewPage.tsx

```diff
@@ -167,6 +172,37 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         });
     }
 
+    fetchCosmicData(mutations:Mutation[]):Promise<ICosmicData> {
+        const generateMap = function(cosmicMutations:CosmicMutation[]) {
+            // key => geneSymbol_proteinPosition
+            // protienPosition => start[_end]
+            const map: ICosmicData = {};
+
+            // create a map for a faster lookup
+            _.each(cosmicMutations, function(cosmic:CosmicMutation) {
+                if (!map[cosmic.keyword]) {
+                    map[cosmic.keyword] = [];
+                }
+
+                map[cosmic.keyword].push(cosmic);
+            });
+
+            return map;
+        };
+
+        const queryKeywords:string[] = _.uniq(_.map(mutations, (mutation:Mutation) => {return mutation.keyword;}));
```

Comment:
you should use the arrow function shortcut syntax:   (mutation:Mutation) => mutation.keyword

if no brackets it knows to return the value